sunpy/map/map_factory.py

```diff
@@ -3,6 +3,7 @@
 import warnings
 import pathlib
 import glob
+from urllib.request import urlopen, Request
```

Comment:
```suggestion
from urllib.request import Request
```

Reply:
I think this is still needed in `_is_url`?

Reply:
My find and search must have been playing up.

Reply:
Seriously, i can't find it.

Reply:
https://github.com/sunpy/sunpy/pull/4037/files#diff-9306ac2bb55014e8e339ffdedaab50c5L347